src/generators/scripts/start.js

```diff
@@ -0,0 +1,70 @@
+import { startScriptBash } from '../bashHelper'
+import { addScriptExtension, scriptHeader } from './utils'
+import { isWin32 } from '../../utils/execUtils'
+
+export default {
+  filename: addScriptExtension('start'),
+  executable: true,
+  generate: (config) => {
+    switch (config.network.deployment) {
+      case 'bash':
+        return startScriptBash(config)
+      case 'docker-compose':
+        return startScriptDocker()
+      case 'kubernetes':
+        return isWin32() ? startScriptKubernetesWindows() : startScriptKubernetesBash()
+      default:
+        return ''
+    }
+  },
+}
+
+function startScriptDocker() {
+  return `${scriptHeader()}
+docker-compose up -d`
+}
+
+function startScriptKubernetesBash() {
+  return `${scriptHeader()}
+
+# check kubectl is installed
+kubectl > /dev/null 2>&1
+EXIT_CODE=$?
+
+if [ $EXIT_CODE -ne 0 ];
+then
+  printf "Error: kubectl not found, please install kubectl before running this script.\n"
+  printf "For more information, see our qubernetes project: https://github.com/jpmorganchase/qubernetes\n"
+  exit $EXIT_CODE
+fi
+
+kubectl cluster-info >nul 2>&1
```

Comment:
when i restart docker and don't have minikube running, this command hangs. 
but then when i start up minikube and then stop it, the command is fine.

Reply:
good catch, that's the windows version of that command. I just pushed a fix